packages/dso-toolkit/src/components/document-component/document-component.args.ts

```diff
@@ -162,11 +172,27 @@ export const documentComponentArgTypes: ArgTypes<DocumentComponentArgs> = {
     control: {
       type: "text",
     },
+    if: { arg: "mode", eq: "document" },
   },
   enableRecursiveToggle: {
     control: {
       type: "boolean",
     },
+    if: { arg: "mode", eq: "document" },
+  },
+  mode: {
+    defaultValue: "document",
```

Comment:
`defaultValue` is deprecated: https://storybook.js.org/docs/api/arg-types#defaultvalue

Reply:
Aha